docs/troubleshoot/minimal.md

```diff
@@ -76,15 +76,39 @@ Migration progress can be tracked by monitoring the Dataflow job and following c
 
 It can happen that in retryDLQ mode, there are still permanent errors. To identify that all the retryable errors have been processed and only permanent errors remain for reprocessing - one can look at the ‘Successful events' count - it would remain constant after every retry iteration. Each retry iteration, the ‘elementsReconsumedFromDeadLetterQueue' would increment.
 
-### Retry command
+### Retry commands
+
+#### To rerun regular flow
+
+To rerun the regular flow, the same command as original needs to be fired. Note: This will only work when not using the PubSub subscriptions for GCS files.The processing starts all over again, meaning the same Datastream outputs get reprocessed.
+
+```
+gcloud dataflow flex-template run <jobName> \
+ --project=<project-name> --region=<region-name> \
+ --template-file-gcs-location=gs://dataflow-templates-southamerica-west1/2023-09-12-00_RC00/flex/Cloud_Datastream_to_Spanner \
+ --num-workers 1 --max-workers 50 \
+ --enable-streaming-engine \
+ --parameters databaseId=<database id>,deadLetterQueueDirectory=<GCS location of the DLQ directory>,inputFilePattern=<gcs location of the datastream output>,instanceId=<spanner-instance-id>,sessionFilePath=<GCS location of the session json>,streamName=<data stream name>,transformationContextFilePath=<path to transformation context json>
+
+```
+
+These job parameters can be taken from the original job.
+
+#### To reprocess DLQ directory
```

Comment:
Also mention that to run in dlq retry mode, add `runMode=retryDLQ` to the parameters.

It is not easy to see whats different by default.

Reply:
Actually I've added that to the sample command. So user does not need to add it manually